src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -521,7 +521,7 @@ void ClassLoader::setup_app_search_path(JavaThread* current, const char *class_p
 
   while (cp_stream.has_next()) {
     const char* path = cp_stream.get_next();
-    update_class_path_entry_list(current, path, false, false, false);
+    update_class_path_entry_list(current, path, true, false, false);
```

Comment:
Some comments on these boolean parameters would be useful.

Reply:
I added some comments to the above and another callsite in the same file.